errbot/botplugin.py

```diff
@@ -239,10 +244,15 @@ def stop_poller(self,
     def program_next_poll(self,
                           interval: float,
                           method: Callable[..., None],
+                          times: int=None,
                           args: Tuple=None,
                           kwargs: Mapping=None):
+        if type(times) in [int, float] and times <= 0:
```

Comment:
don't check the type, if it is a string there between crashing and failing silently, it is way better that it crashes ! :)

just: 
```python
if times is not None and times <= 0:
```